bigtable/google/cloud/bigtable/row_data.py

```diff
@@ -425,18 +451,9 @@ def _copy_from_previous(self, cell):
                 cell.row_key = previous.row_key
             if not cell.family_name:
                 cell.family_name = previous.family_name
-            # NOTE: ``cell.qualifier`` **can** be empty string.
-            if cell.qualifier is None:
+            if not cell.qualifier:
```

Comment:
This is a reversion, see #4252.

Did you revert this on purpose or do you have a 2+ month old checkout of the codebase?